contracts/contracts/RequestManager.sol

```diff
@@ -706,4 +711,33 @@ contract RequestManager is Ownable, LpWhitelist, CrossDomainRestrictedCalls {
         require(deprecated == false, "Contract already deprecated");
         deprecated = true;
     }
+
+    /// Mark the request identified by ``requestId`` as filled by ``filler``.
+    ///
+    /// .. note::
+    ///
+    ///     This function is callable only by the native L2 messenger contract,
+    ///     which simply delivers the message sent from L1 by the
+    ///     Beamer's L2 :sol:interface:`messenger <IMessenger>` contract.
```

Comment:
I would remove this entire note as it is no longer correct.

Reply:
why is this no longer correct?

Reply:
Same thing as in https://github.com/beamer-bridge/beamer/pull/1032#discussion_r973053687.